packages/performance/wafer-map/src/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Wafer Map Performance Tests</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <meta name="description" content="Wafer map performance tests">
+    <script type="module" src="./index.ts"></script>
+</head>
+<body>
+    <p>See devtools performance timing for results.</p>
+    <p>Loading...</p>
+    <p class="running" hidden>Running...</p>
+    <p class="finished" hidden>Finished!</p>
```

Comment:
As we are cleaning up after the tests I added these status messages so it is easier to know when the tests are running or finihsed.